### ISSUE TITLE
Box sizing modifiers

### ISSUE THREAD

**Issue by JakeWharton** [JakeWharton/mosaic#11]
The ability to specify how large a box is explicitly.

This should clip if you exceed the size. Otherwise you're padded with spaces (as is the default anyway).

**Comment by JakeWharton** [comment]
Leaving this open because we can do it now, but it will likely be subsumed into #8.

**Comment by JakeWharton** [comment]
This is unblocked now.

**Comment by EpicDima** [comment]
I would like to clarify what else is currently expected within this issue?

**Comment by JakeWharton** [comment]
Probably nothing! Let's close. Anything else specific can get its own issue.
